services/stripe-webhook.php

```diff
@@ -228,6 +228,7 @@
 			$old_order = getOldOrderFromInvoiceEvent($pmpro_stripe_event);
 			$user_id = $old_order->user_id;
 			$user = get_userdata($user_id);
+			$user->membership_level = pmpro_getMembershipLevelForUser($user_id);
```

Comment:
```suggestion
```
This change is no longer needed as the user's membership level is now being retrieved in the sendBillingFailureEmail() method as a result of this PR:
https://github.com/strangerstudios/paid-memberships-pro/pull/1586